modules/microshift-cni-multus-bridge-example-config.adoc

```diff
@@ -0,0 +1,233 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-cni-multus-bridge-example-config_{context}"]
+= Example configuration for the bridge additional network
+
+Use the following procedure to configure a bridge-type additional network. You can use the example workflow here to configure other supported additional network types.
+
+[IMPORTANT]
+====
+The minor version of {microshift-short} and the version of the {microshift-short} Multus CNI plugin must match. You must use the 4.16 version of both RPMs, for example.
+====
+
+.Prerequisites
+
+* The Multus CNI is installed.
+* The OpenShift CLI (`oc`) is installed.
+* The cluster is running.
+
+[IMPORTANT]
+====
+The following example configuration uses the `bridge` additional network type. When using `bridge` and the `dhcp` IPAM, a DHCP server listening on the bridge interface is required. If you are using a firewall, configuring the firewalld service to allow DHCP traffic on the interface or network zone is also required.
+====
+
```

Comment:
I think we should either explain here how this is done, or point to where the information can be found ("see above bridge interface specifics")

Reply:
noted